internal/controller/streams.go

```diff
@@ -543,13 +570,47 @@ func ConnectStreams(ctx context.Context, logger *zap.SugaredLogger, envMap Envir
 				failureReason = "Log stream closed"
 			}
 			authConContextCancel()
+			logger.Warn("One or more streams have been closed; restarting all streams", zap.String("reason", failureReason))
```

Comment:
Why was it needed to duplicate the log below?
https://github.com/illumio/cloud-operator/pull/162/files#diff-65d4066e2022c168a97a2083474d348fa6d5d1fffb6e8d7378282935ad187940R590
```suggestion
```